src/components/Form/ProfileForm.tsx

```diff
@@ -31,29 +31,36 @@ const validationSchema = Yup.object({
   title: Yup.string().required('title is required'),
 });
 
-function ProfileForm({ callback }: { callback?: () => void }) {
+function ProfileForm({
+  callback,
+  editedUser,
```

Comment:
could be nice if in the 2 case we pass the user we want to edit as argument. It would lead to cleaner code with no 2 cases to managed inside here